pom.xml

```diff
@@ -1031,5 +1091,48 @@ limitations under the License.
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>release</id>
```

Comment:
I think we should do the javadoc build pre-commit if we need for release. Otherwise, I think we'll have to chase it at release time if javadoc fails (for example because of not using &lt; as in this changeset.

Reply:
I was wondering about this too